vim.exe.stackdump

```diff
@@ -0,0 +1,19 @@
+Stack trace:
+Frame        Function    Args
+00180238000  0018005D19E (00180223639, 00180223C39, 001802342F0, 000FFFFB720)
+00180238000  001800463F9 (CCCCCC00009CC1, FF783B00767676, D6D661000CC616, 9E00B4005648E7)
+00180238000  00180046432 (00180223616, 000000001E7, 001802342F0, 76767600CCCCCC)
+00180238000  001800431E3 (00000000000, 00180238000, 7FFAA8DC888E, 001800004EC)
+00180238000  0018006B101 (CCCCCC00009CC1, FF783B00767676, D6D661000CC616, 9E00B4005648E7)
+00180238000  0018006BF4C (00000000000, 0010065D548, 00000000000, 00000000000)
+00180238000  0018006E066 (00000000000, 00000000008, 005FCB3E280, 00000000000)
+00000000001  00180135376 (0010065D540, 00000000008, 00000000000, 00000000000)
+00000000001  0018011C6FB (0010065D540, 00000000008, 00000000000, 00000000000)
+00000000001  001004F77E4 (00100575E0A, 00000000000, 00000000001, 00100663D0C)
+00000000001  001005820F3 (00000000008, 00100664140, 00000000000, 00000000000)
+00000000001  00100576D86 (00000000000, 0005AD97CA3, 0002160892C, 00000010000)
+00000000001  001005D4CEB (000FFFFCCB0, 00000000000, 0018011C6FB, 00000000001)
+006000491F0  001005E218A (00000000020, 001802E0900, 00180047986, 00180046990)
+000FFFFCCB0  001800479F7 (00000000000, 00000000000, 00000000000, 00000000000)
+00000000000  00180045663 (00000000000, 00000000000, 00000000000, 00000000000)
+End of stack trace (more stack frames may be present)
```

Comment:
Did you mean to include this?

Reply:
No, this was a mistake -- will delete this.

Reply:
Deleted